ddtrace/tracer/span_test.go

```diff
@@ -522,6 +522,17 @@ func BenchmarkSetTagString(b *testing.B) {
 	}
 }
 
+func BenchmarkSetTagStringer(b *testing.B) {
+	span := newBasicSpan("bench.span")
+	keys := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	value := &stringer{}
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		k := string(keys[i%len(keys)])
```

Comment:
Is the key important to the benchmark? If so, why?

Reply:
This benchmark is a simple copy of `BenchmarkSetTagString`. (the one above)
Should I simplify it, and use a static key ?

Reply:
Ahh, I see now it's a copy of `BenchmarkSetTagString`. That's fine then. 👍